changelogs/fragments/3084-info-checkmode.yaml

```
@@ -0,0 +1,13 @@
+minor_changes:
```

These are clearly bugs.
```suggestion
bugfixes:
```